apps/a11y-tests/src/axe-sarif-converter/check-message-transformer.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { CheckConfiguration, FormattedCheckResult } from './ruleresults';
+
+export class CheckMessageTransformer {
```

Comment:
Don't quite understand why this is a class--does the code where it's used require a class instance?

Reply:
Never mind since this is temporary